velox/functions/lib/string/tests/StringImplTest.cpp

```diff
@@ -529,6 +543,10 @@ TEST_F(StringImplTest, pad) {
   runTestUserError("text", -1, "a");
   runTestUserError(
       "text", ((int64_t)std::numeric_limits<int32_t>::max()) + 1, "a");
+  // Additional tests with bad unicode bytes.
+  runTest("abcd\uFFFFef", 6, "0", "abcd\uFFFFe", "abcd\uFFFFe");
```

Comment:
should not 6 trim it at 6 chars? hence output would be "abcd\uFFFF" with out the e. 
or does \u have different meaning here?
https://fburl.com/daiquery/5wrzojzv

Reply:
Its because \uFFFF is valid unicode char : https://www.fileformat.info/info/unicode/char/ffff/index.htm . The actual bad unicode is \xFF . I will change comment above in case thats confusing.